src/views/product-tutorials-view/helpers/__tests__/detect-and-reformat-learn-url.test.ts

```diff
@@ -67,7 +67,7 @@ describe('detectAndReformatLearnUrl', () => {
 			// Note: underlying rewriteTutorialsLink() handles beta products
 			{
 				input: '/consul',
-				expected: 'https://learn.hashicorp.com/consul',
```

Comment:
💭 We may want to actually update this one so its using a non beta product still if we want to keep the test 100% the same. 
```
/vagrant
https://learn.hashicorp.com/vagrant
```

Reply:
Great call! Updated in 51180eaae2ca39b38396e8451d4830f296fec62c